app/forms/PropertyDetailsFormsValidation.scala

```diff
@@ -136,13 +156,13 @@ object PropertyDetailsFormsValidation {
     }
   }
 
-
+//todo - refector the method below
```

Comment:
Is this something you're able to do as part of this PR?

Reply:
I don't actually think this needs refactoring.